code-challenges.test.js

```diff
@@ -23,12 +26,51 @@ var colors1 = ["purple", "blue", "green", "yellow", "pink"]
 var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+// write out jest test with the description beggining with the functions name i will be using later 
+// write in varialbe arrays which holds strings inside  in the test in order to make the test pass 
+//put in expected outcomes 
+//variables let color1 
+//          let color2 
+
+//got the test written out for boths problems. 
+//needing help with the logic for them 
+describe ("removeAndRandom", () => {
+  let colors1 = ["purple", "blue", "green", "yellow", "pink"]
+  let colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+  it (" removes the first item from the array and shuffles",()=> {
+    expect(removeAndRandom(colors1)).toEqual(
+//write the logic that will shift and shuffle colors 1
+
+    )
+    expect(removeAndRandom(colors2)).toEqual(
 
+        //write the logic that will shift and shuffle colors 2
+    )
+  })
+} )
 
 
 
 // b) Create the function that makes the test pass.
+ //write a function that will go through the array remove the first index  and then randomize the array 
 
+ // trying to use the map highorder function to go through the array.
+ // then using the built in methods shift to remove the first index and shuffle to randomize them 
+//shift(colors1) shuffle(colors1)
```

Comment:
Okay this was a solid attempt. I would have liked to see this attempt made and reviewed and thought about if we had more than a weekend. Keep practicing the logic and problem solving thats needed for coding. "What are the smallest steps I can break a problem down too to solve?" "Am I assuming anything that I don't know is real?" "What can I look up to write this code?"